Bottom: 748417186433c05a057ad0f0be8897d08922ce3c
Top:    8c833f2e28c02872bfe6a3cc7c2be1fe9913d56a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 15:12:32 +0000

Refresh of disable-debug

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 489004b..67d98f5 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1505,7 +1505,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               NULL));
 
 
-    break;
+    //break;
   }
